docs/guide/ingress/annotations.md

```diff
@@ -12,6 +12,7 @@ You can add annotations to kubernetes Ingress and Service objects to customize t
     - Annotations that configures LoadBalancer / Listener behaviors have different merge behavior when IngressGroup feature is been used. `MergeBehavior` column below indicates how such annotation will be merged.
         - Exclusive: such annotation should only be specified on a single Ingress within IngressGroup or specified with same value across all Ingresses within IngressGroup.
         - Merge: such annotation can be specified on all Ingresses within IngressGroup, and will be merged together.
+    - The controller may not update the configuration of an *existing* load balancer if an annotation is added/updated. New load balancers will be have an appropriate configuration. 
```

Comment:
This controller should update the existing load balancer. In certain case, for example when the load balancer scheme gets modified from internal to internet-facing or vice-versa, controller deletes existing lb and recreates a new one.

Reply:
I commented my issue on the other PR (https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/2680#discussion_r925156540) -- let's discuss there